src/convertToObject.js

```diff
@@ -12,7 +12,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cascadingStyleSheets = {};
+  const stringConverted = sourceString.split(';');
+  const splittedByDoubledots = stringConverted.map(style => style.split(':'));
+  const filtered = splittedByDoubledots.filter(style => style.length === 2);
+  const deletedGaps = filtered.map(
+    ([key, value]) => [key.trim(), value.trim()]
+  );
```

Comment:
Роби відступи між змінними
```suggestion
  const cascadingStyleSheets = {};
  
  const stringConverted = sourceString.split(';');
  
  const splittedByDoubledots = stringConverted.map(style => style.split(':'));
  
  const filtered = splittedByDoubledots.filter(style => style.length === 2);
  
  const deletedGaps = filtered.map(
    ([key, value]) => [key.trim(), value.trim()]
  );
```